### PR TITLE
Implement unescape helper and style fixes.

### PR DESCRIPTION
**Pull request by mattcollier**
Although the test suite is passing, I don't know if this C++ code is doing the right thing, or if it's implemented properly, but I figured I'd get the ball rolling.

Just commented regex code so comparison would be easier, this stuff should be removed.

I found the benchmarks too variable to be useful, see: https://gist.github.com/mattcollier/33613cef6565c22cb78e84e20e34d260

Lifted the unescape code from here: https://stackoverflow.com/a/5612287

### CODE REVIEW

**Comment by mattcollier** [comment]
Added a benchmark that shows a 100% performance increase with the `unescape` implementation. 
 See results: https://gist.github.com/mattcollier/d01faf4e2f2be59f9d8796523e7f3a50

**Comment by dlongley** [comment]
@davidlehn, pulling this in would yield a very significant performance increase -- can you see if we have proper testing for it and add a test or two if not?

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "browser": true,
+    "mocha": true,
+    "phantomjs": true
```
Comment (davidlehn) [inline .eslintrc]: This package isn't using phantomjs.
```suggestion
```

File: benchmark/benchmark.js [hunk 2]
```diff
@@ -37,16 +37,16 @@ if(!TEST_SUITE) {
...
+  // 'rdfn:Urgna2012EvalTest': {
+  //   params: [
+  //     parseNQuads(readTestNQuads('action')),
+  //     createTestOptions({
+  //       algorithm: 'URGNA2012',
+  //       inputFormat: 'application/nquads',
+  //       format: 'application/nquads'
+  //     })
+  //   ]
+  // },
```
Comment (davidlehn) [inline benchmark/benchmark.js:49]: Is this supposed to be commented out?

**Comment by davidlehn** [comment]
- I'm not sure what extra testing is needed.  I'm guessing it'd be some crazy escaping edge case. Would be in the main test suite if so.  Any thoughts on oddball patterns that could cause trouble?
- If we're not using regexes then can clean up that file: remove regex include, remove regexes, remove commented out code, move iostream header up, remove other unused commented old things.
- I think that regex call was the only reason we needed all that goofy travis compiler update code.  Would be nice to not need that!

**Comment by mattcollier** [comment]
@davidlehn please make whatever changes are necessary here and wrap it up.

That commented about test suite stuff was not running properly, it doesn't seem to be a thing in the current test suites if you download a new copy of it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
 # rdf-canonize-native ChangeLog
 
+### Fixed
+- **BREAKING**: N-Quad canonical serialized output.
+  - Only escape 4 chars.
+
+### Changed
+- N-Quad native serialization optimization.
+  - Varies based on input by rougly ~1-5x.
```
Comment (dlongley) [inline CHANGELOG.md]:
```suggestion
  - Varies based on input by roughly ~1-5x.
```
